src/OpenTelemetry/Trace/Configuration/OpenTelemetrySdk.cs

```diff
@@ -106,5 +100,35 @@ public static void EnableOpenTelemetry(Action<OpenTelemetryBuilder> configureOpe
 
             ActivitySource.AddActivityListener(listener);
         }
+
+        internal static void BuildSamplingParameters(
+            in ActivityCreationOptions<ActivityContext> options, out ActivitySamplingParameters samplingParameters)
+        {
+            ActivityContext parentContext = options.Parent;
+            if (options.Parent == default)
+            {
+                // Check if there is already a parent for the current span.
+                var parentSpan = Activity.Current;
```

Comment:
I wonder if .NET Activity API can take care of this part - if user did not explicitly use activitySource.StartActivity(name, parentcontext/id), can it automatically use Activity.Current?
@tarekgh please share your thoughts on this.

Reply:
Can you clarify more here? The current APIs, if you pass the parent context == default, then we'll use Activity.Current as the parent for the newly created Activity. What exactly the ask here?

Reply:
Tarek and I chatted offline and clarified this.

Essentially the ask is, when doing StartActivity() without explicitly passing parentContext, can ActivitySource implicitly take Activity.Current.Context. 
(This is what Paulo is doing here, but the ask is to move it into .NET itself)

Reply:
Thanks for your clarification. I'll look at that.